webapp/src/pxtjson.tsx

```diff
@@ -68,12 +77,12 @@ export class Editor extends srceditor.Editor {
                 }
             }
             // trigger update            
-            update(uc);
+            save();
         }
         return (
             <div className="ui content">
                 <div className="ui segment form text" style={{ backgroundColor: "white" }}>
-                    <sui.Input label={lf("Name")} value={c.name} onChange={v => update(c.name = v)} />
+                    <sui.Input label={lf("Name")} value={c.name} onChange={(v) => updateName(c.name = v)} />
```

Comment:
we should not use the onChange, it is broken for script languages. Can we do without it?